### ISSUE TITLE
Methicillin-resistant Staphylococcus aureus infection

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6069]
**Preferred term label:**
Methicillin-resistant Staphylococcus aureus infection

**Synonyms**
MRSA

**Definition (free text, please give PubMed ID)**
Methicillin resistance has occurred in S. aureus by mutation of a penicillin-binding protein, a chromosome-encoded protein. MRSA can cause a range of organ-specific infections, the most common being the skin and subcutaneous tissues, followed by invasive infections like osteomyelitis, meningitis, pneumonia, lung abscess, and empyema.
PMID: 29489200

**Parent term (use hpo.jax.org/app)**
Recurrent Staphylococcus aureus infections

**Comment by pnrobinson** [comment]
This term already exists, https://hpo.jax.org/app/browse/term/HP:0032326
